cli/commands.go

```diff
@@ -71,7 +71,7 @@ func Commands(metaPtr *meta.Meta) map[string]cli.CommandFactory {
 				CredentialBackends: map[string]logical.Factory{
 					"approle":  credAppRole.Factory,
 					"cert":     credCert.Factory,
-					"aws-ec2":  credAwsEc2.Factory,
```

Comment:
TBC, I originally had both `aws-ec2` and `aws` here without the alias code, but I changed it to using the alias at your suggestion... :)